CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 * Fixes page metadata for `partition_html` when `include_metadata=False`
 * A `ValueError` now raises if `file_filename` is not specified when you use `partition_via_api`
   with a file-like object.
+* Re-enable test_upload_label_studio_data_with_sdk
```

Comment:
this line looks good, but please add the entry to the top of the changelog with an incremented -devX version (and then also bump the same -devX version in unstructured/_version_.py)